python/paddle/fluid/contrib/slim/tests/qat_int8_comparison.py

```diff
@@ -209,7 +221,7 @@ def _predict(self,
                 samples = len(data)
                 total_samples += samples
                 batch_times.append(batch_time)
-                fps = samples / batch_time
+                fps = samples / batch_time * 1000
```

Comment:
Our goal for performance is set to compare with optimized FP32 MKL-DNN model, so what do you think if we compare the performance and accuracy results with the our transformed FP32 model?

Reply:
For now, the comparison to original QAT model allows for better determination of accuracy changes introduced by quantization and passes applied. It also provides a great insight into how do the application of passes and running of real int8 kernels accelerates the whole process. I propose to keep it the way it is for now.